src/core_plugins/kibana/public/dashboard/panel/panel_utils.js

```diff
@@ -78,7 +85,15 @@ export class PanelUtils {
   static parseVersion(version = '6.0.0') {
     const versionSplit = version.split('.');
     if (versionSplit.length < 3) {
-      throw new Error(`Invalid version, ${version}, expected <major>.<minor>.<patch>`);
+      throw new Error(i18n.translate('kbn.dashboard.panel.invalidVersionErrorMessage', {
+        defaultMessage: 'Invalid version, {version}, expected {major}.{minor}.{patch}',
```

Comment:
I think we can put `<major>.<minor>.<patch>` in a one variable. The order will not be changes.